ci-operator/step-registry/ipi/conf/ipi-conf-commands.sh

```diff
@@ -33,4 +33,18 @@ if [ ${FIPS_ENABLED} = "true" ]; then
 	cat >> "${out}" << EOF
 fips: true
 EOF
-fi
\ No newline at end of file
+fi
+
+if [ ${RT_ENABLED} = "true" ]; then
```

Comment:
This will overwrite the `install-config`. Also, all the data are incorrect. I am not sure how and where we should use the `MachineConfig`